notebooks/MAST/TESS/interm_tess_prf_retrieve/tess_prf_retrieval.ipynb

```diff
@@ -0,0 +1,852 @@
+    "    path = path + addPath\n",
+    "    \n",
+    "    #Convert the fractional pixels to the offset required for the interleaved pixels.\n",
+    "    colOffset, rowOffset = getOffsetsFromPixelFractions(col, row)\n",
+    "    \n",
+    "    #Determine the 4 (col,row) locations with exact PRF measurements.\n",
+    "    imagePos = determineFourClosestPrfLoc(col, row)\n",
+    "    bestPos = imagePos[0]\n",
+    "    prfArray = readOnePrfFitsFile(ccd, camera, bestPos[0], bestPos[1], path, datestring)\n",
+    "            \n",
+    "    prfImage = getRegSampledPrfFitsByOffset(prfArray, colOffset, rowOffset)       \n",
+    "\n",
+    "    return prfImage"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def getPrfAtColRowFits(col, row, ccd, camera, sector, path):\n",
+    "    \"\"\"\n",
+    "    Main Function\n",
+    "    Lookup a 13x13 PRF image for a single location\n",
+    "    \n",
+    "    Inputs\n",
+    "    ---------\n",
+    "    col, row\n",
+    "        (floats) Location on CCD to lookup. The origin of the CCD is the bottom left.\n",
+    "        Increasing column increases the \"x-direction\", and row increases the \"y-direction\"\n",
+    "        The column coordinate system starts at column 45.\n",
+    "    ccd\n",
+    "        (int) CCD number. There are 4 CCDs per camera\n",
+    "    camera\n",
+    "        (int) Camera number. The instrument has 4 cameras\n",
+    "    sector\n",
+    "        (int)  Sector number, greater than or equal to 1.\n",
+    "    path\n",
+    "        (str) Directory or URL where the PRF fits files are located\n",
+    "\n",
+    "    Returns\n",
+    "    ---------\n",
+    "    A 13x13 numpy image array of the interpolated PRF.\n",
+    "    \"\"\"\n",
+    "    col = float(col)\n",
+    "    row = float(row)\n",
+    "    prfImages = []\n",
+    "    \n",
+    "    #Determine a datestring in the file name and the path based on ccd/camera/sector\n",
+    "    datestring, subDirectory = pathLookup(ccd, camera, sector)\n",
+    "    path = path + subDirectory\n",
+    "    \n",
+    "    #Convert the fractional pixels to the offset required for the interleaved pixels.\n",
+    "    colOffset, rowOffset = getOffsetsFromPixelFractions(col, row)\n",
+    "    \n",
+    "    #Determine the 4 (col,row) locations with exact PRF measurements.\n",
+    "    imagePos = determineFourClosestPrfLoc(col, row)\n",
+    "\n",
+    "    #Loop over the 4 locations and read in each file and extract the sub-pixel location.\n",
+    "    for pos in imagePos:\n",
+    "            prfArray = readOnePrfFitsFile(ccd, camera, pos[0], pos[1], path, datestring)\n",
+    "            \n",
+    "            img = getRegSampledPrfFitsByOffset(prfArray, colOffset, rowOffset)       \n",
+    "            prfImages.append(img)\n",
+    "\n",
+    "    #Simple linear interpolate across the 4 locations.\n",
```

Comment:
#Simple -> # Simple

I'm going to stop pointing these out now, but all the comments that start with no space should have a space.

Reply:
Some day I will figure out why there is a standard one whether there are spaces or not.  Until then, I'll keep adding spaces all over the place.